File: DataStructures/LinkedList/IterativeLength.java
class Node 
{
    int data;
    Node next;
    Node(int d)
    {
        data = d;
        next = null;
    }
}

class LinkedList
{
    Node head;

    public void push(int new_data)
    {
        Node new_node = new Node(new_data);
        new_node.next = head;
        head = new_node;
    }


    public int getCount(Node head)
    {
        int count = 0;
        Node current = head;
        while(current != null)
        {
            current = current.next;
            count = count + 1;
        }
        return count;
    }

    public static void main(String[] args)
    {
        LinkedList llist = new LinkedList();
        llist.push(1);
        llist.push(12);
        llist.push(13);
        llist.push(14);

        System.out.print("Count of nodes is " + llist.getCount());
        
    }
}

File: DataStructures/LinkedList/IterativeLength.py
class Node:
    def __init__(self, data):
        self.data = data
        self.next = None

class LinkedList(self):
    def __init__(self):
        self.head = None

    def push(self, new_data):

        new_node = Node(new_data)
        new_node.next = self.head
        self.head = new_node

    def getCount(self):
        current = self.head
        count = 0
        while(current):
            current = current.next
            count = count + 1
        return count

if __name__ == '__main__':
    llist = LinkedList()
    llist.push(1)
    llist.push(2)
    llist.push(3)
    llist.push(4)
    print("Count of nodes is: ", llist.getCount())